SignalAnalysis/localization/strings.tr.resx

```diff
@@ -0,0 +1,522 @@
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="resheader">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" />
+            </xsd:complexType>
+          </xsd:element>
+        </xsd:choice>
+      </xsd:complexType>
+    </xsd:element>
+  </xsd:schema>
+  <resheader name="resmimetype">
+    <value>text/microsoft-resx</value>
+  </resheader>
+  <resheader name="version">
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="strBtnAccept" xml:space="preserve">
+    <value>&amp;aktif</value>
+  </data>
+  <data name="strBtnCancel" xml:space="preserve">
+    <value>&amp;iptal</value>
+  </data>
+  <data name="strBtnData" xml:space="preserve">
+    <value>Veri seçin</value>
+  </data>
+  <data name="strBtnExport" xml:space="preserve">
+    <value>&amp;İhracat</value>
+  </data>
+  <data name="strBtnReset" xml:space="preserve">
+    <value>&amp;Sıfırla</value>
+  </data>
+  <data name="strBtnSettings" xml:space="preserve">
+    <value>&amp;Ayarlar</value>
+  </data>
+  <data name="strChkCrossHair" xml:space="preserve">
+    <value>Arsaların artı işaretini göster</value>
+  </data>
+  <data name="strChkCumulative" xml:space="preserve">
+    <value>Kümülatif fraktal boyut</value>
+  </data>
+  <data name="strChkDlgPath" xml:space="preserve">
+    <value>Aç/kaydet iletişim kutusunu önceki yolu hatırla</value>
+  </data>
+  <data name="strChkEntropy" xml:space="preserve">
+    <value>Entropi (yaklaşık ve örnek)</value>
+  </data>
+  <data name="strChkPower" xml:space="preserve">
+    <value>Güç (dB)</value>
+  </data>
+  <data name="strDlgReset" xml:space="preserve">
+    <value>Tüm alanları sıfırlamak istiyor musunuz?
+varsayılan değerlerine?</value>
+  </data>
+  <data name="strDlgResetTitle" xml:space="preserve">
+    <value>Ayarları Sıfırla?</value>
+  </data>
+  <data name="strErrorDeserialize" xml:space="preserve">
+    <value>Ayarlar dosyası yüklenirken hata oluştu.
+
+{0}
+
+Bunun yerine varsayılan değerler kullanılacaktır.</value>
+  </data>
+  <data name="strErrorDeserializeTitle" xml:space="preserve">
+    <value>Hata</value>
+  </data>
+  <data name="strFileHeader00" xml:space="preserve">
+    <value>ErgoLux verileri</value>
+  </data>
+  <data name="strFileHeader01" xml:space="preserve">
+    <value>Sinyal Analizi verileri</value>
```

Comment:
Could it be 'SinyalAnalizi verileri' (without the space between signal and analysis)?

Besides that, is it 'SinyalAnalizi' or 'SinyalAnaliz' (as you suggested in the open/save dialog filters (see comments below)?

Reply:
SinyalAnalizi verileri